diagnostic.md

```diff
@@ -56,13 +70,29 @@ let answer = 'not empty';
 
 while (answer !== '') {
   answer = ask("Guess my secret? ");
-  if (answer === 'SeCrEt') {
+  if (answer === '7') {
    console.log("You guessed it!");
+   else if (answer <1 && >10){
```

Comment:
You are missing another "answer" in this check. It should be (answer <1 && answer >10)